tests/subyt/test_source_config.py

```
@@ -0,0 +1,111 @@
+        assert found_names[2].value == "Marc"
+        found_ages = list(
+            g.objects(predicate=URIRef("https://example.org/peeps/age"))
+        )
+        assert len(found_ages) == 3
+        assert found_ages[0].value == 40
+        assert found_ages[1].value == 48
+        assert found_ages[2].value == 54
+
+
+@pytest.mark.parametrize("source_param", [CONFIG_DICT, CONFIG_STRING])
+def test_api_with_string(source_param) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `source_param`

(ANN001)